flink-connectors/flink-connector-kinesis/src/test/java/org/apache/flink/streaming/connectors/kinesis/util/KinesisConfigUtilTest.java

```diff
@@ -277,7 +377,7 @@ public void testIllegalValueForInitialTimestampInConfig() {
 		testConfig.setProperty(ConsumerConfigConstants.STREAM_INITIAL_POSITION, "AT_TIMESTAMP");
 		testConfig.setProperty(ConsumerConfigConstants.STREAM_INITIAL_TIMESTAMP, "-1.0");
 
-		KinesisConfigUtil.validateConsumerConfiguration(testConfig);
+		KinesisConfigUtil.validateConsumerConfiguration(testConfig, new ArrayList<>());
```

Comment:
@xiaolong-sn consider overloading the method for backwards compatibility

Reply:
I've overloaded this method.